package.json

```diff
@@ -75,11 +78,11 @@
     "exports-loader": "^0.6.4",
     "extract-text-webpack-plugin": "^2.1.2",
     "file-loader": "^0.11.2",
-    "gotrue-js": "^0.9.3",
```

Comment:
To be consistent with what we have right now, I moved `gotrue-js` into the regular dependencies.